.travis.yml

```diff
@@ -21,9 +21,9 @@ deploy:
     secure: X0tQXL0stFL2cE+PHM6ZoqDHSgHQEQPJ/uD5FZJanueZ/w9Q2QxVYg9fYj3ITBKZX6DEd95Qsnfx8rwnMf8LJG+snDPhv0oCUWuZAgtaz3W59teAA195nNTkWWrEnIiR1eWZo/apxImSHfdJmjdvhcMYHDAidalNVMOwcTNgDvwmAC55S3CI9mhVqE/7zTJHTWyz4p+tWYlHJTzMTNQMfKrcbYzT9pyWUpKcgRtFDOBdq5Vp/AJahT3NajU4G+UoVOIzq8iDB+q81fnIF8TbV/RrUFB5fLHkYGBo4bw0UOoM2kH7og9Xe6T0GOgBT84bCYQ+V+xkCJIPs/d8/w1dm5fBtzJKsHjFHMDEan2oDNEAN1B1o7sZ0mpVpbBc1/caYLYXJ6GO8L2VpnXFHBOPzOEWi7HPLL5ydn0KQHUR9YeX2jyFZmlhXd7+xlRSHqZIPC0zGQ8JStvz9z5lx02/Oq/1fmRLPf1i8dWmAJkIgTzFa2FRCVhzQZ/uHD64paF81mcOeVq6y/swcoYvj+G+CNyBnAMn1Cgu6eelsHtnr3TzwROHCLz3pFhp2IL32O+Awt/f8F3OaOa+SNzWtHn6XW8rtsVrgWj8yki6AqntRs3fuOBTyUReJRLuyPjCfbVjGNKNljX7lf53WUWD/eAlQa6pAOjtz97DGi1VkTAQer4=
```

Comment:
@josejulio now we have a new repo (kiali/kiali-ui) you will need to run the "travis encrypt" tool to get a new key.
We also need @kiali/kiali-ui project created in npm repo.

Reply:
Already did that and tested locally :)

Reply:
Just to clarify, the same key should work for the new org, is my key.

Reply:
OK, I'll merge and we'll see.